builder/FindMFX.cmake

```diff
@@ -39,18 +39,12 @@ else()
 endif()
 
 find_path( MFX_INCLUDE mfxdefs.h PATHS ${MFX_API_HOME}/include NO_CMAKE_FIND_ROOT_PATH )
```

Comment:
It seems that there should be (or could be?) a part in this file which is not needed if we take dispatcher from the sources... I.e. the part protected by `if (PKG_MFX_FOUND)`. Maybe that's not for this change, but I would feel more comfortable having this part clearly separated.